scripts/prow-test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+npm install jest@26.6.3
+npm install react-tippy@1.4.0
+npm install jsoneditor@9.6.0
+npm run test-shared-lib
```

Comment:
Run the tests in here as well 

- `compass-console/compass` (npm run test)
- `compass-console/components/react` (npm run test)